program-workflows/hiv-recency-testing.md

```diff
@@ -117,3 +117,40 @@ https://github.com/METS-Programme/hie-documentation/blob/master/reference/recenc
    * 401 authentication error: This implies that either the recency server URL, username and or password are incorrect. Double check these values and also verify the credentials in Mirth.
 3. Reduce the scheduled sync frequency to 10 minutes, restart UgandaEMR and observe the behaviour.
 
+# Upload Recency Data to the Portal
+
+## SOP for How to Download the Data export CSV from the EMR and Upload onto the Recency Portal
+
+### HOW TO DOWNLOAD THE DATA EXPORT CSV FROM THE EMR
+
+1. While on the Home screen page, click the “UgandaEMR REPORTS – 2019 HMIS TOOLS” button to be able to download the right report.
+   ![Reports_Module](/assets/reports_module.png)
```

Comment:
@ODORA0 

1. Please make all image file names lower case
2. Use a relative path to the images so that they are easily accessed